spark/dl/pom.xml

```diff
@@ -51,7 +51,7 @@
         <dependency>
             <groupId>com.intel.analytics.bigdl.native</groupId>
             <artifactId>${mkl-java-os-version}</artifactId>
-            <version>0.1.0</version>
+            <version>0.2.0-SNAPSHOT</version>
```

Comment:
Currently, we should modify the bigdl.native to 0.2.0-SNAPSHOT in spark/dl/pom.xml because of changed BigDL-core.